project/api/models/account.py

```diff
@@ -259,3 +261,28 @@ def is_full_account(self):
             return True
         else:
             return False
+
+    def get_voted_bills(self):
+        from .activity import Activity
+
+        activities = Activity.objects.filter(account=self).prefetch_related(
+            'civi__linked_bills', 'civi__linked_civis__linked_bills'
+        )
+        supported_bills = []
+        opposed_bills = []
+
+        for activity in activities.iterator():
+            if activity.is_voted_negatively:
```

Comment:
Can we rename this to `activity.is_negative_vote` / `activity.is_positive_vote`? It is a bit less verbose.